tox.ini

```diff
@@ -38,5 +41,5 @@ commands = pylint scality_sproxyd_client
 
 [flake8]
 # E501 line too long
-ignore = E501
+ignore = E501,E722
```

Comment:
Catching specific exceptions is recommended when possible. 

You should not ignore it IMHO.